### PR TITLE
Update distro version in __init__.py

### PR DESCRIPTION
**Pull request by mishrahrishikesh**
Update the bundled package distro from 1.6.0 to 1.9.0 in __init__.py (lib/ansible/module_utils/distro/__init__.py)

##### SUMMARY

Latest package of distro has been released and it has been changed from 1.6.0 to 1.9.0 in __init__.py file.

Fixes #83114

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

Component Name: lib/ansible/module_utils/distro/__init__.py

```paste below

```

### CODE REVIEW

File: lib/ansible/module_utils/distro/__init__.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 from __future__ import annotations
 
 # The following makes it easier for us to script updates of the bundled code
-_BUNDLED_METADATA = {"pypi_name": "distro", "version": "1.6.0"}
+_BUNDLED_METADATA = {"pypi_name": "distro", "version": "1.9.0"}
```
Comment (Akasurde) [inline lib/ansible/module_utils/distro/__init__.py]:
```suggestion
_BUNDLED_METADATA = {"pypi_name": "distro", "version": "1.8.0"}
```

Comment (Akasurde) [reply lib/ansible/module_utils/distro/__init__.py]: The current version of https://github.com/python-distro/distro is set to 1.8.0.

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Review by Akasurde — APPROVED** [review]

**Comment by mattclay** [comment]
This was a follow-up to https://github.com/ansible/ansible/pull/81765, which updated `distro` to 1.8.0 without updating `_BUNDLED_METADATA` to match.

**Comment by Akasurde** [comment]
Thanks @mattclay.

**Pull request merged**